lab_1_classify_by_unigrams/main.py

```diff
@@ -34,7 +43,14 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    if not isinstance(tokens, list) or None in tokens\
+            or not all(isinstance(token,str) for token in tokens):
+        return None
+    frequency_dictionary = {}
+    for token in tokens:
+        if token not in frequency_dictionary:
+            frequency_dictionary[token] = tokens.count(token)/len(tokens)
```

Comment:
It is unoptimize. Can you try to call `.count()` only for unique symbols

Reply:
done